VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleListEntryController.cs

```diff
@@ -59,19 +62,8 @@ public class CatalogModuleListEntryController : CatalogBaseController
         [ResponseType(typeof(webModel.ListEntrySearchResult))]
         public IHttpActionResult ListItemsSearch(webModel.SearchCriteria criteria)
         {
-            var coreModelCriteria = criteria.ToCoreModel();
-            ApplyRestrictionsForCurrentUser(coreModelCriteria);
-
-            coreModelCriteria.WithHidden = true;
-
-            // need search in children categories if user specify keyword
-            if (!string.IsNullOrEmpty(coreModelCriteria.Keyword))
-            {
-                coreModelCriteria.SearchInChildren = true;
-                coreModelCriteria.SearchInVariations = true;
-            }
 
-            var result = _listEntrySearchService.Search(coreModelCriteria);
+            var result = _listEntrySearchService.Search(PrepareSearchCriteria(criteria));
```

Comment:
I think you could make method SearchListEntries instead of PrepareSearchCriteria with `_listEntrySearchService.Search` call

Reply:
done